Mathematics/Calculus_and_Analysis/Real_Analysis 2.tex

```diff
@@ -7,7 +7,7 @@
 \end{theorem} 
 \begin{proof}
     (1) $\Rightarrow$ (2)\qquad $ \left|L-S_n\right| = \left|\lim_{m\to\infty} S_m - S_n\right| = \left|\lim_{m\to\infty} \sum_{k=n+1}^m a_k\right|\\ = \left|\sum_{k=n+1}^\infty a_k\right| < \epsilon$. $\hfill\blacksquare$ \vspace{1em}\\
-    (2) $\Rightarrow$ (3) \qquad $\left|\sum_{k=n+1}^\infty a_k\right| = \left|\sum_{k=n+1}^\infty a_k - \sum_{k=m+1}^\infty a_k\right| < \frac{\epsilon}{2} + \frac{\epsilon}{2} = \epsilon$.  $\hfill\blacksquare$ \vspace{1em}\\
+    (2) $\Rightarrow$ (3) \qquad $\left|\sum_{k=n+1}^\infty a_k\right| = \left|\sum_{k=n+1}^\infty a_k - \sum_{k=m+1}^\infty a_k\right| < \cfrac{\epsilon}{2} + \cfrac{\epsilon}{2} = \epsilon$.  $\hfill\blacksquare$ \vspace{1em}\\
```

Comment:
Unfortunately, `\cfrac` will make the length between lines weird. Sorry for the confusion. 
You may still keep the `\frac\` here ^-^. I would change it for this file. But keep this in mind.

Perhaps,
1. Math mode in sentences, i.e, $..$, you may use `\frac` if it is a/b, but `\frac{a/b}{c} if it is a/b/c.
2. Math mode in math environment, i.e, 
\[
     ....
\] 
then you may use `\frac` if a/b, and `\cfrac` if a/b/c.